docs: Add symbolicate-js endpoint documentation (#1208)
getsentry/symbolicator: docs/api/sourcemaps.md

```http
POST /symbolicate-js?timeout=123&scope=123 HTTP/1.1
Content-Type: application/json

{
  "source": {
    "id": "<id>",
    "url": "https://sentry.io/api/0/projects/sentry-org/sentry-project/artifact-lookup/",
    "token": "secret"
  },
  "stacktraces": [
    {
      "frames": [
        {
          "function": "x",
          "module": "x",
          "filename": "x",
          "abs_path": "x",
          "lineno": 1,
          "colno": 1,
          "in_app": true
        }
      ]
    }
  ],
  "modules": [
    {
      "code_file": "some-code-file",
      "debug_id": "some-debug-id",
      "type": "debug_id"
    }
  ],
  "dist": "production",
  "release": "1.33.1",
  "scraping": {
    "enabled": true,
    "allowed_origins": ["*.domain.com"]
  }
}
```

## Query Parameters

- `timeout`: If given, a response status of `pending` might be sent by the
  server.
- `scope`: An optional scope which will be used to isolate cached files from
  each other

## Request Body

    pub scraping: ScrapingConfig,
    pub options: JsRequestOptions,

A JSON payload describing the stack traces and code modules for symbolication,
as well as configuration for scraping sources from external servers:

> - `source`: A descriptors for Sentry source to be used for symbolication. See

Reviewer: ```suggestion
- `source`: A descriptor for the Sentry source to be used for symbolication. See
```

  [Sentry](index.md) source.
- `modules`: A list of source code files with a corresponding debug id that
  were loaded during JS code execution. The list is handled by the Sentry source.
- `stacktrace`: A list of stacktraces to symbolicate.
  - `frames`: A list of frames with corresponding `abs_path`, `lineno`,

>     and `colno`, as well as minified `function` name.

Reviewer: Strictly speaking `colno` is optional.

>     and `colno`, as well as minified `function` name.

Author: ```suggestion
    and other optional fields like `colno` or minified `function` name.
```
